validation/src/main/scala/hmda/validation/ValidationStats.scala

```diff
@@ -0,0 +1,61 @@
+package hmda.validation
+
+import akka.actor.{ ActorRef, ActorSystem, Props }
+import hmda.model.fi.SubmissionId
+import hmda.persistence.messages.CommonMessages.{ Command, Event, GetState }
+import hmda.persistence.model.HmdaPersistentActor
+
+object ValidationStats {
+
+  def name = "ValidationStats"
+  case class SubmissionStats(id: SubmissionId, totalLars: Int)
+  case class AddSubmissionStats(stats: SubmissionStats) extends Command
+  case class SubmissionStatsAdded(stats: SubmissionStats) extends Event
+  case class FindTotalLars(institutionId: String, period: String) extends Command
+
+  def props(): Props = Props(new ValidationStats)
+
+  def createValidationStats(system: ActorSystem): ActorRef = {
+    system.actorOf(ValidationStats.props(), "validation-stats")
+  }
+
+  case class ValidationStatsState(stats: Seq[SubmissionStats] = Nil) {
+    def updated(event: Event): ValidationStatsState = event match {
+      case SubmissionStatsAdded(s) =>
+        ValidationStatsState(stats :+ s)
+    }
+  }
+}
+
+class ValidationStats extends HmdaPersistentActor {
+  import ValidationStats._
+
+  override def persistenceId: String = s"$name"
+
+  var state = ValidationStatsState()
+
+  override def updateState(event: Event): Unit = {
+    state = state.updated(event)
+  }
+
+  override def receiveCommand: Receive = super.receiveCommand orElse {
+    case AddSubmissionStats(stats) =>
+      persist(SubmissionStatsAdded(stats)) { e =>
+        log.debug(s"Persisted: $stats")
+        updateState(e)
+      }
+    case FindTotalLars(id, period) =>
+      val submissionStats = state.stats
+      val filtered = submissionStats.filter(s => s.id.institutionId == id && s.id.period == period)
+      if (filtered.nonEmpty) {
+        val submission = filtered.sortWith(_.id.sequenceNumber > _.id.sequenceNumber).head
+        sender() ! submission.totalLars
+      } else {
+        sender() ! 0
```

Comment:
Do we want to default to 0 here?  If an Institution doesn't have a submission from the previous year, it will cause Q011 to fail no matter how many LARs are in their current submission.  (This might also be a question for @lesgou)

For example, all of the current `clean_X-lars.txt` files will trigger Q011 when running the API in demo mode.